packages/turf-union/index.ts

```diff
@@ -7,8 +7,7 @@ import { Feature, Polygon, MultiPolygon, Properties } from "@turf/helpers";
  * Takes two {@link (Multi)Polygon(s)} and returns a combined polygon. If the input polygons are not contiguous, this function returns a {@link MultiPolygon} feature.
  *
  * @name union
- * @param {Feature<Polygon|MultiPolygon>} polygon1 input Polygon feature
- * @param {Feature<Polygon|MultiPolygon>} polygon2 Polygon feature to difference from polygon1
+ * @param {Array<Feature<Polygon|MultiPolygon> | Polygon | MultiPolygon>} features Input features to be unioned
```

Comment:
Is there any reason we don't make it a `FeatureCollection` rather than array? I think more traditionally we've used standard geojson arguments. 
Not going to die in a ditch about it though :)

Reply:
Yeah I'll rework this to take a FeatureCollection to be consistent.